File: Solutions/3403/c++/program.cpp
// Copy from Quera

#include <iostream>
#include <iomanip>
using namespace std;

int main()
{
    double a, b, c, d;
    cin >> a >> b >> c >> d;
    cout << setprecision(6) << fixed << "Sum : " << (a + b + c + d) << endl;
    double ave = a + b + c + d;
    cout << setprecision(6) << fixed << "Average : " << ave / 4 << endl;
    cout << setprecision(6) << fixed << "Product : " << 1ll * a * b * c * d << endl;
    cout << setprecision(6) << fixed << "Max : " << max(a, max(b, max(c, d))) << endl;
    cout << setprecision(6) << fixed << "Min : " << min(a, min(b, min(c, d))) << endl;
    return 0;
}

File: Solutions/3403/java/program.java
// Copy from Quera

import java.util.Scanner;
import java.lang.Math;
public class Main {
    static Scanner s=new Scanner(System.in);
    public static void main(String[] args) {
        
        int a = s.nextInt(), b = s.nextInt(), c = s.nextInt(), d = s.nextInt();
        
        double Sum = a + b + c + d;
        double Average = (double)(a + b + c + d) / 4;
        double Product = (a * b * c * d);
        double Max = Math.max(Math.max(a, b), Math.max(c, d));
        double Min = Math.min(Math.min(a, b), Math.min(c, d));
        
        System.out.printf("Sum : %.6f\n", Sum);
        System.out.printf("Average : %.6f\n", Average);
        System.out.printf("Product : %.6f\n", Product);
        System.out.printf("Max : %.6f\n", Max);
        System.out.printf("Min : %.6f\n", Min);
    }
}